### PR TITLE
feat: add api "M1-21 resolveFrontOfIdentityCard" & "M1-20 resolveBackOfIdentityCard" & "M1-22resolveBussinessLicense" & "M1-10 personalCertificate" & "M1-11 companyCertificate"

### PR DESCRIPTION
**Pull request by Ranklier-Hu**
Signed-off-by: hujiahui hujiahui@yuhu.tech

### CODE REVIEW

File: util/httplib.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"strings"
+)
+
+func SetUrlParam(url string, data url.Values) string {
+	urlStr := url + "?"
+	paramStrs := make([]string, len(data))
+
+	i := 0
+	for key, value := range data {
+		paramStr := key + "=" + value[0]
```
Comment (chenqing-yuhu) [inline util/httplib.go]: 使用strings.Builder来拼接字符串性能最好

File: prisma/datamodel.graphql [hunk 2]
```diff
@@ -178,3 +191,22 @@ type MessageContent {
...
+type Article {
+	id: ID! @unique
+	createdAt: DateTime!
+	title: String
+	source: String
+	articleURL: String
+	imgURL: String
+}
+
+type Erc20 {
```
[the remark this answers is not in the document]
Comment (Ranklier-Hu) [reply prisma/datamodel.graphql]: 已经将此表删除

File: resolver/address_resolver.go [hunk 3]
```diff
@@ -17,3 +17,7 @@ func (a *Address) AddressString(ctx context.Context) *string {
 func (a *Address) Nickname(ctx context.Context) *string {
 	return a.Address.Nickname
 }
+
+func (a *Address) Erc20s(ctx context.Context) (*[]*Erc20, error) {
+	panic("not implemented")
```
[the remark this answers is not in the document]
Comment (Ranklier-Hu) [reply resolver/address_resolver.go]: 已经将此字段删除

File: resolver/mutation_resolver.go [hunk 4]
```diff
@@ -262,20 +297,85 @@ func (r *Resolver) ReceiveInvitionReward(ctx context.Context, args *struct {
...
+}
+
+func (r Resolver) UpdateArticle(ctx context.Context, args *struct {
+	ID         string
+	Title      *string
+	Source     *string
+	ArticleURL *string
+	ImgURL     *string
+}) (*Article, error) {
+	if args.Title != nil && *args.Title == "" {
```
[the remark this answers is not in the document]
Comment (Ranklier-Hu) [reply resolver/mutation_resolver.go]: 已经将逻辑改成 if args.Title == nil || *args.Title == ""

File: util/httplib.go (hunk 1 above)
Comment (Ranklier-Hu) [reply util/httplib.go]: 已经使用strings.Builder来拼接字符串

**Pull request merged**